sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java

```diff
@@ -244,6 +295,12 @@ private void fillDateOfSymptoms(LocalDate dateOfSymptom) {
         DATE_OF_ONSET_INPUT, DATE_FORMATTER.format(dateOfSymptom));
   }
 
+  private void setSpecificSymptoms(String parameter) {
```

Comment:
the method sets a specific symptom, but inside covers 3, which are not all. Can we adjust it a bit? or maybe move out each field to make it reusable